src/helpers/relative-activity.tsx

```diff
@@ -0,0 +1,75 @@
+import { add } from "date-fns";
+import { ColorDefinition } from "./colors";
+import { queryDailyData } from "./query-daily-data";
+import getDayKey from "./get-day-key";
+
+export interface RelativeActivityQueryResult {
+    relativePercent: number;
+    value: number;
+    threshold: number;
+}
+
+export interface RelativeActivityDataType {
+    dailyDataType: string;
+    color?: ColorDefinition;
+    overThresholdColor?: ColorDefinition;
```

Comment:
It would probably be nice to support "under" thresholds as a first class option as well.  SpO2 and Steps come to mind as data types for which a low value is more likely to be highlighted.

I suppose you could achieve that by making the base color the highlight color and the `overThresholdColor` the normal color.

Reply:
Exactly, that's just swapping the colors essentially, like with sleep you could highlight under 7.5 hours or something